Source/Analytics/Read/Overview/LastWeeksPerHealthRisk/CaseReportsLastWeeksForHealthRisk.cs

```diff
@@ -0,0 +1,14 @@
+using Concepts;
+using Concepts.HealthRisks;
+
+namespace Read.Overview.LastWeeksPerHealthRisk
+{
+    public class CaseReportsLastWeeksForHealthRisk
```

Comment:
Would ..Last4Weeks...  or last MonthDividedInWeeks or something like that be a better name? How do I know how many seeks I get?

Reply:
You get no seeks. But yes, I guess "LastFourWeeks" or "Last4Weeks" depending on taste might be better. Last month is a bit ambiguous since the number of weeks is not always four.